meta-balena-common/recipes-core/initrdscripts/files/migrate

```diff
@@ -96,7 +101,9 @@ migrate_enabled() {
 migrate_run() {
     # Find the raw image in the rootfs partition
     image=$(find "${ROOTFS_DIR}" -xdev -type f -name "${BALENA_IMAGE}")
-    kernel_images=$(find "${ROOTFS_DIR}" -xdev -type f -name "@@KERNEL_IMAGETYPE@@*")
+    kernel_images=$(find "${ROOTFS_DIR}" -xdev -type f -name "Image*")
+    jetson_orin_capsule=$(find "${ROOTFS_DIR}" -xdev -type f -name "TEGRA_BL_*Cap*")
+    jetson_orin_boot_artifact=$(find "${ROOTFS_DIR}" -xdev -type f -name "boot0.img.gz")
```

Comment:
These are device specific - we need to find a generic way to hook into this script from a device repository and not hard code device specifics in meta-balena.

Maybe calling out to extra_device_artifacts() or something like that.